docker-compose_developer.yml

```diff
@@ -87,6 +112,9 @@ services:
 
   vpw-polling-client:
     image: public.ecr.aws/viadee/camunda-kafka-polling-client:latest
+    profiles:
```

Comment:
so "polling-client-local" means we do not fire up a polling client with docker-compose and instead start the respective spring-boot-app in order to develop that piece of software further?

Reply:
Yes, exactly.